torch/_refs/nn/functional/__init__.py

```diff
@@ -156,3 +158,57 @@ def softplus(
         rhs = refs.log1p(refs.exp(scaled_input))
 
     return refs.where(refs.gt(scaled_input, threshold), a, rhs)
+
+
+# Losses
+def _apply_loss_reduction(loss: TensorLikeType, reduction: str) -> TensorLikeType:
+    if reduction == "sum":
+        return refs.sum(loss)
+    elif reduction == "mean":
+        return refs.true_divide(refs.sum(loss), loss.numel())
```

Comment:
cc @ngimel who just added the mean reduction, which is probably what we want to use here? 

Neat workaround, though